src/rust/cryptography-x509-validation/src/ops.rs

```diff
@@ -73,8 +73,10 @@ zl9HYIMxATFyqSiD9jsx
 
         let ops = NullOps {};
         assert_eq!(ops.public_key(&cert), Ok(()));
-        assert!(ops
-            .verify_signed_by(&cert, ops.public_key(&cert).unwrap())
-            .is_ok());
+        assert!({
+            ops.public_key(&cert).unwrap();
+            ops.verify_signed_by(&cert, ())
+        }
+        .is_ok());
```

Comment:
What's going on here, looks like the public_key call is a no-op?

Reply:
From a quick look, this was probably just to get coverage on the `NullOps` test fixture. This can now probably be removed outright!

Reply:
af14ec426